scripts/pre_commit_linter.py

```diff
@@ -988,12 +990,19 @@ def _check_docstrings(all_files):
             for line_num in range(file_length):
                 line = file_content[line_num].lstrip().rstrip()
                 prev_line = ''
+                pattern_to_match = r'^""".+"""$'
```

Comment:
@kevinlee12 why would we want a docstring with no content?

Reply:
Hmm, I thought we had a case where we handled it inside the if-else block here, but a second glance says otherwise. You are free to leave it as-is then.